### PR TITLE
removes dextrous_hud_type and healable var

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

I was working on some basic mob stuff and noticed this is a little messy, so I made these changes separately.
I removed ``dextrous_hud_type`` because ``hud_type`` exists and can just be used instead.
I also removed the ``healable`` var, because it was incorrectly used (with the expectation that basic mobs had it too). Instead it will rely on the mob being Organic, the check right under it, and I gave mob biotypes to mobs that were not healable and had no biotype already.
I made a new biotype for slimes because I didn't find any other ones that fit it (and gave it to slimepeople), but it is not used anywhere other than to prevent them from healing from sutures, as I didn't want to just set it to NONE. Thought this may be useful for the future.

## Why It's Good For The Game

``healable`` is checked on simple animals and basic mobs, despite basic mobs not having this var. I do not want to add this var to basic mobs either, I think checking for them being organic makes much more sense and avoids having to make basic mobs less basic.

## Changelog

Nothing player-facing.

### CODE REVIEW

**Comment by timothymtorres** [comment]
Code janis doing god's work.

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm [hunk 1]
```diff
@@ -207,6 +207,7 @@
 	emote_see = list("honks", "sweats", "jiggles", "contemplates its existence")
 	speak_chance = 5
 	dextrous = TRUE
+	hud_type = /datum/hud/dextrous
```
Comment (Ghommie) [inline code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm:210]: Likely on accident, but you're giving the dexterous hud to fleshclowns. 🤡 🍖

**Review by Ghommie — APPROVED** [review]
just fix the fleshy whoopsie and conflicts and it should be ready to go. EDIT: Nevermind, it is not a whoopsie.

File: code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm:210]: They are apparently supposed to have it, from the ``dextrous`` var above being true

Comment (Ghommie) [reply code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm:210]: Oh, you're right. I totally glossed over it.

**Pull request merged**